cosmoz-treenode-button-view.html

```diff
@@ -44,7 +47,7 @@
 		</style>
 		<div class="horizontal layout">
 			<paper-button class="flex" raised on-tap="openDialogTree">
-				<div class="pathToNode">[[ _getButtonLabel(nodesOnNodePath, buttonTextPlaceholder) ]]</div>
+				<div class="pathToNode" has-nodes$="[[ nodesOnNodePath.length ]]">[[ _getButtonLabel(nodesOnNodePath, buttonTextPlaceholder) ]]</div>
```

Comment:
This works, but does not look  really "adaptive".
What about having a `<div>&nbsp</div>`, inside the `paper-button` hidden when nodePath is set ?